draft-ietf-quic-transport.md

```diff
@@ -3612,10 +3612,15 @@ When new data is to be sent on a stream, a sender MUST set the encapsulating
 STREAM frame's offset field to the stream offset of the first byte of this new
 data.  The first octet of data on a stream has an offset of 0.  An endpoint is
 expected to send every stream octet.  The largest offset delivered on a stream
-MUST be less than 2^62. A receiver MUST ensure that received stream data is
-delivered to the application as an ordered byte-stream.  Data received out of
-order MUST be buffered for later delivery, as long as it is not in violation of
-the receiver's flow control limits.
+MUST be less than 2^62.
+
+A receiver MUST ensure that received stream data is delivered to the application
```

Comment:
I think Patrick wanted to allow out of order delivery, and I thought the agreement was along the lines of "A receiver MUST provide the received stream data to the application as an ordered byte-stream".  The text you wrote implies it's in-order is the only possible option.

Reply:
That text isn't changed.  It's what we litigated (at some length), but moved into a new paragraph, because the old one was getting unwieldy.

You are right, though.  It seems to imply that it has to deliver in order, when the intent was that it be capable of doing so.